docs/yaml/functions/response_file.yaml

```diff
@@ -0,0 +1,71 @@
+name: response_file
+returns: list[str] | str
+description: |
+  Write argument list to a response file, if the resulting
+  length of the command line is longer than a maximum value.
+
+  Returns either the list of arguments or the path to the response
+  file.
+
+  This function is designed to be used with `custom_target`,
+  `run_target`, `run_command`, `test`, or any other function
+  running a custom executable or script.
+
+varargs:
+  name: arguments
+  type: str | file | list
+  description: The argument list.
+
+kwargs:
+  prefix:
+    type: str
+    default: "'@'"
+    description: |
+      The prefix to prepend to the response file path.
+      Use an empty string if you don't need a prefix.
+  
+  quote:
+    type: str | bool
+    default: true
+    description: |
+      If `true`, quote the arguments in the response file
+      using the default quote function, based on the operating
+      system.
+
+      If `false`, do not quote the arguments.
+
+      Can also be `'sh'` to use sh-like quoting,
+      `'cmd'` for Windows cmd-like quoting,
+      or `'gcc'` for gcc-like quoting.
+
+  max_args_length:
+    type: int | bool
+    default: 127
```

Comment:
Where does the 127 come from? Is that a Windows thing?

Reply:
https://learn.microsoft.com/en-us/cpp/build/reference/at-specify-a-compiler-response-file?view=msvc-170#remarks